viewing_party/party.py

```diff
@@ -1,23 +1,152 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    new_movie = {}
+    new_movie["title"] = title
+    new_movie["genre"] = genre
+    new_movie["rating"] = rating
+    
+    if None in new_movie.values():
+        return None
+    else:
+        return new_movie

```

Comment:
This looks great! One nice thing about python is that a function will return None by default if it isn't given an explicit return value, so we could just have:

     if None not in new_movie.values():
          return new_movie

And we wouldn't even need an else statement.